test/complex.jl

```diff
@@ -1088,8 +1088,8 @@ end
         @test isequal(one(T) / complex(T(-NaN),  T(-Inf)), complex(-zero(T), zero(T)))
 
         # divide complex by complex Inf
-        @test isequal(complex(one(T)) / complex(T(Inf), T(-Inf)), complex(zero(T), zero(T))) broken=(T==Float64)
-        @test isequal(complex(one(T)) / complex(T(-Inf), T(Inf)), complex(-zero(T), -zero(T))) broken=(T in (Float32, Float64))
+        @test isequal(complex(one(T)) / complex(T(Inf), T(-Inf)), complex(zero(T), zero(T)))
+        @test isequal(complex(one(T)) / complex(T(-Inf), T(Inf)), complex(zero(T), -zero(T)))
```

Comment:
Why the sign change?

Reply:
Because I was having trouble with math. Complex numbers in IEEE don't have a multiplicative identity. `Complex(1.,0.)*Complex(-0.,-0.)==Complex(0.,-0.)` and  `Complex(1.,-0.)*Complex(-0.,-0.)==Complex(-0.,0.)`, so my idea for getting the signs wasn't working...

Reply:
I think it should be the following: in `(a + b * im) / (c + d * im)`, assuming `a` and `b` finite and non-zero, and `c` and `d` infinite:

* real part: `sign(a) * sign(c)`
* imaginary part: `sign(b) * -sign(d)`

Reply:
For completeness, you should likely refer to the [ISO/IEC 10967-3:2006](https://standards.iso.org/ittf/PubliclyAvailableStandards/c037994_ISO_IEC_10967-3_2006(E).zip) standard

Reply:
It's a bit worrying to me how e.g. signs in tests are being changed with an off-hand "fix tests" commit. It's to me a bit of a [Chesterton's fence](https://fs.blog/chestertons-fence/) situation. A significant amount of work has gone in to making sure that these things are working the way they are and one needs to be quite careful when changing these pretty low-level operations.

Reply:
@giordano thanks for finding the thing that actually works.